### PR TITLE
render policy sample check_deprecated_apis

### PR DESCRIPTION
**Pull request by JimBugwadia**
Signed-off-by: Jim Bugwadia <jim@nirmata.com>

### CODE REVIEW

File: content/en/policies/best-practices/check_deprecated_apis.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Check Deprecated Apis"
+category: Best Practices
+version: 
+subject:
```
Comment (chipzoller) [inline content/en/policies/best-practices/check_deprecated_apis.md:5]: We need a subject for this so it'll be accessible to a filter, and I'm not sure it's best to list all of these resources by name in a comma-separated list. Ideas?

Comment (JimBugwadia) [reply content/en/policies/best-practices/check_deprecated_apis.md:5]: How about using `Kubernetes API` as the subject?

I'll also fix the title...

Comment (chipzoller) [reply content/en/policies/best-practices/check_deprecated_apis.md:5]: I was kind of thinking something along those lines, so I think that'll work. Hopefully the filter logic doesn't break when it encounters a space (not sure that was tested).

**Comment by JimBugwadia** [comment]
Works like a charm! https://deploy-preview-213--kyverno.netlify.app/policies/best-practices/check_deprecated_apis/?policytypes=Kubernetes%2520APIs

**Review by chipzoller — APPROVED** [review]

**Pull request merged**